### PR TITLE
Initial commit

### PR DESCRIPTION
**Pull request by jplindstrom**

### CODE REVIEW

File: source/dist.ini [hunk 1]
```diff
@@ -0,0 +1,52 @@
+name = Time-Progress-Stored
```
Comment (keithbro) [inline source/dist.ini:1]: Wow

Comment (jplindstrom) [inline source/dist.ini:1]: I know, right?

File: source/lib/Time/Progress/Stored/Storage.pm [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+=head1 METHODS
+
+=head2 store($id, $content) : Bool
+
+Store the current report $content (a data structure) under the $id
+key.
+
+=cut
+
+=cut
```
Comment (keithbro) [inline source/lib/Time/Progress/Stored/Storage.pm]: extra cut?

**Comment by keithbro** [comment]
I approve! (is there a button for it?)

**Pull request closed without merging**